frontend/hooks/useServiceTemplates.ts

```diff
@@ -3,8 +3,12 @@ import { SERVICE_TEMPLATES } from '@/constants/serviceTemplates';
 
 export const useServiceTemplates = () => {
   const getServiceTemplates = (): ServiceTemplate[] => SERVICE_TEMPLATES;
-  const getServiceTemplate = (hash: string): ServiceTemplate | undefined =>
-    SERVICE_TEMPLATES.find((template) => template.hash === hash);
+  const getServiceTemplate = (
+    serviceUuid: string,
+  ): ServiceTemplate | undefined =>
+    SERVICE_TEMPLATES.find(
+      (template) => template.service_config_id === serviceUuid,
+    );
 
```

Comment:
this won't work :/

service template shouldn't contain a service_config_id, only service responses from middleware
service template is what we send to middleware as the template used to build the service

we could do it by `hash` though, but need to make sure we check the new hash history array, incase the service template we have is newer than the one currently used by a deployment

can leave for now as most desirable changes are the API types, and can refactor in layer PR though